src/components/mobileservices/BindButton.test.js

```diff
@@ -0,0 +1,55 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import BindButton from './BindButton';
+
+const setup = (propOverrides = {}) => {
+  const defaultProps = {
+    buildConfig: {
+      repoUrl: 'http://example.com',
+      branch: 'example-branch',
+      jobName: 'job-11010',
+      jenkinsfilePath: 'https://jenkins.file/path'
+    },
+    service: {
+      isBindingOperationInProgress: () => false
+    },
+    onClick: jest.fn()
+  };
+
+  const props = { ...defaultProps, ...propOverrides };
+  const wrapper = shallow(<BindButton {...props} />);
+
+  return {
+    props,
+    wrapper
+  };
+};
+
+describe('BindButton Good', () => {
+  const { wrapper } = setup();
+
+  it('should render component', () => {
+    expect(wrapper.find('Button').hasClass('bind-button')).toEqual(true);
+    expect(wrapper.find('Button').contains('Bind to App')).toEqual(true);
+  });
+});
+
+describe('onClick()', () => {
+  const { wrapper } = setup();
+
+  it('should press button', () => {
+    const saveButton = wrapper.find('Button');
+
+    expect(saveButton.simulate('click')).toHaveLength(1);
+  });
+});
+
+describe('BindButton Bad', () => {
```

Comment:
This should be more descriptive. Something like:

'Binding operation in progress'